### PR TITLE
 ADAPT - converted to es6 and did some tidy up around show correct answer

### PR DESCRIPTION
**Pull request by eleanor-heath**
Contributing towards OS code: adaptlearning/adapt_framework#2808  
"Convert all core components to ES6 classes #2808"

Converted to es6 and did some tidy up around show correct answer.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-slider",
-  "version": "3.0.0",
-  "framework": ">=5",
+  "version": "3.0.1",
+  "framework": ">=5.7.0",
```
Comment (moloko) [inline bower.json:4]: I _think_ `">=5.5"` would be OK here... but I might have missed something?

File: js/sliderView.js [hunk 2]
```diff
@@ -3,33 +3,35 @@ define([
...
-      'click .js-slider-number-click': 'onNumberSelected',
-      'focus .js-slider-item-input': 'onHandleFocus',
-      'blur .js-slider-item-input': 'onHandleBlur'
-    },
+    events() {
+      return {
+        'click .js-slider-number-click': 'onNumberSelected',
+        'focus .js-slider-item-input': 'onHandleFocus',
+        'blur .js-slider-item-input': 'onHandleBlur'
+      }
```
Comment (moloko) [inline js/sliderView.js:13]:
```suggestion
      };
```

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: js/sliderView.js [hunk 3]
```diff
@@ -289,74 +291,77 @@ define([
...
       this.selectItem(userAnswerIndex, true);
       this.animateToPosition(this.mapIndexToPixels(userAnswerIndex));
       this.setSliderValue(this.model.get('_userAnswer'));
-    },
+    }
 
     // according to given item index this should make the item as selected
-    selectItem: function(itemIndex, noFocus) {
-      _.each(this.model.get('_items'), function(item, index) {
+    selectItem(itemIndex, noFocus) {
```
Comment (moloko) [inline js/sliderView.js:363]: the parameter `noFocus` isn't actually used so can be removed... there's a few calls to this function which might need updating as well
```suggestion
    selectItem(itemIndex) {
```

**Comment by eleanor-heath** [comment]
Closing this PR and opening https://github.com/adaptlearning/adapt-contrib-slider/pull/146

**Pull request closed without merging**